### ISSUE TITLE
BUG New-ADChangeReport CSS Path not detected.

### ISSUE THREAD

**Issue by jdhitsolutions** [jdhitsolutions/ADReportingTools#24]
This is a bug reported by @klaypigeon

```powershell
PS C:\Windows\system32> new-adchangereport
You cannot call a method on a null-valued expression.
At C:\Program Files\WindowsPowerShell\Modules\ADReportingTools\1.0.0\Functions\ADChangeReport.ps1:130 char:13
+             $htmlParams.Add("CSSPath", $CSSPath)
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : InvokeMethodOnNull
```

**Comment by jdhitsolutions** [comment]
I had re-organized some code and I see now that I am trying to add something that hasn't been defined yet. This should be easy to fix.

**Comment by jdhitsolutions** [comment]
This should be fixed in v1.1.0 (38ccd83). Update the module from the PowerShell gallery and try again.

**Comment by klaypigeon** [comment]
Thanks. That was some mighty fast bug fixing.
